edxval/tests/constants.py

```diff
@@ -268,6 +268,22 @@
     subtitles=[SUBTITLE_DICT_SRT],
     **VIDEO_DICT_STAR
 )
+COMPLETE_SET_WITH_COURSE_KEY = dict(
+    courses=['edX/DemoX/Demo_Course'],
+    encoded_videos=[
+        ENCODED_VIDEO_UPDATE_DICT_STAR
```

Comment:
Might be better to use "ENCODED_VIDEO_DICT_STAR" since UPDATE was originally meant to be used when updating an object.